paasta_tools/contrib/ide_helper.py

```diff
@@ -18,12 +18,28 @@ def merge_vscode_settings(file: str, setting_dict: Dict[str, Any]) -> None:
         settings = {}
 
     # update settings with the default configurations we want each file to have
-    settings.update(setting_dict)
+    merge_dicts(setting_dict, settings)
 
     with open(json_path, mode="w") as settings_file:
         json.dump(settings, settings_file, indent=2)
 
 
+def merge_dicts(setting_dict: Dict[str, Any], settings: Dict[str, Any]) -> None:
```

Comment:
super-nit: thoughts on this change?
```suggestion
def merge_dicts(merge_from: Dict[str, Any], merge_to: Dict[str, Any]) -> None:
```
would make it a little clearer as to how the merge is meant to work

Reply:
makes sense, thanks!